src/main/java/org/lookout_studios/meals_management_system/meals_management_system/MealsManagementSystemApplication.java

```diff
@@ -3,17 +3,20 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
-
 /*
  * This class handles the app logic.
  * TO-DO: Remove (exclude = { DataSourceAutoConfiguration.class })
  * once the database support is implemented.
  */
-@SpringBootApplication(exclude = { DataSourceAutoConfiguration.class })
+@SpringBootApplication (exclude = { DataSourceAutoConfiguration.class })
 public class MealsManagementSystemApplication {
-
-	public static void main(String[] args) {
-		SpringApplication.run(MealsManagementSystemApplication.class, args);
+	public static void main (String[] args) {
+		SpringApplication.run (MealsManagementSystemApplication.class, args);
+		/* 
+		 * For tests only. 
+		 * TO-DO: Change to desired location when the app logic will be ready. 
+		 */
```

Comment:
This class handles the app logic. Depending on what the method is meant to be doing (it is not possible to figure this out by looking at it), it might be at a right spot